homeassistant/components/schlage/coordinator.py

```diff
@@ -74,3 +77,30 @@ def _get_lock_data(self, lock: Lock) -> LockData:
             LOGGER.debug('Failed to read logs for lock "%s": %s', lock.name, ex)
 
         return LockData(lock=lock, logs=logs)
+
+    @callback
+    def _add_remove_locks(self) -> None:
+        """Add newly discovered locks and remove nonexistent locks."""
+        if self.data is None:
+            return
+
+        self.config_entry: ConfigEntry
```

Comment:
```py
class SchlageDataUpdateCoordinator(DataUpdateCoordinator[SchlageData]):
    """The Schlage data update coordinator."""

    config_entry: ConfigEntry

    def __init__(self, hass: HomeAssistant, username: str, api: Schlage) -> None:
```
Doing this removes the need for this local type overwrite

Reply:
👍🏻 Done.